posts/brugner/1.azuresqledge-docker-m1.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Running Azure SQL Edge on Docker on macOS M1
+published: false
+description: Microsoft hasn't released an ARM-based SQL Server Docker image, what can we do on an Apple M1 Mac if we want to run the popular RDBMS?
+tags: "azuresqledge, docker, m1, arm"
+cover_image: ./assets/cover1.png
+---
+
+In 2020 Apple decided to replace the Intel chipsets with one of their own making, that has been leading performance benchmarks ever since. But these chips are based on the **ARM architecture** instead of the widespread x64 used by its predecesor so, if we want to run software natively and take advantage of all of its power, programs should be ARM ready. This also goes to **Docker** images.
```

Comment:
Who's been `leading performance benchmarks`? The new chipsets or Apple's decision to replace them? If it's the first one, I think `have been` should be used instead 🤔

Reply:
Good catch!